.gitignore

```diff
@@ -24,6 +24,7 @@ dist
 
 # IDE
 .idea
+/*.sublime-*
```

Comment:
This is unrelated to the PR. Ideally, you should use a user-global gitignore locally.

Reply:
Sure, I'll drop it, thanks.